corehq/apps/registration/templates/registration/partials/register_new_user/submit_errors.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+<fieldset class="submit-errors" data-bind="visible: hasSubmitErrors">
+  <legend><i class="fa fa-exclamation-warning-sign"></i> {% trans "Issue with registration" %}</legend>
+  <p>
+    {% blocktrans %}
+    Please check that you have JavaScript enabled to use this site!
+    We found the following issues:
+    {% endblocktrans %}
+  </p>
+  <ul data-bind="foreach: submitErrors">
+    <li><strong data-bind="text: fieldName"></strong> &mdash; <span data-bind="text:errors"></span></li>
+  </ul>
+  <button type="button"
+          class="btn btn-lg btn-primary-dark"
+          data-bind="click: previousStep">{% trans "Go Back & Fix Form" %}</button>
```

Comment:
Biyeun, we may want to stick with just up to naming a project on the first
pass, so as to not have too many variables at play in the A/B test.

So, keeping the confirmation screen a separate webpage for now, running
that as an A/B test, and then adding in the confirmation page change as
another A/B test.

Amelia

On Tue, Jun 21, 2016 at 1:10 PM, Ben <notifications@github.com> wrote:

> In
> corehq/apps/registration/templates/registration/partials/register_new_user/submit_errors.html
> <https://github.com/dimagi/commcare-hq/pull/12165#discussion_r67910303>:
>
> > +{% load i18n %}
> > +
> > +<fieldset class="submit-errors" data-bind="visible: hasSubmitErrors">
> > +  <legend><i class="fa fa-exclamation-warning-sign"></i> {% trans "Issue with registration" %}</legend>
> > +  <p>
> > +    {% blocktrans %}
> > +    Please check that you have JavaScript enabled to use this site!
> > +    We found the following issues:
> > +    {% endblocktrans %}
> > +  </p>
> > +  <ul data-bind="foreach: submitErrors">
> > +    <li><strong data-bind="text: fieldName"></strong> &mdash; <span data-bind="text:errors"></span></li>
> > +  </ul>
> > +  <button type="button"
> > +          class="btn btn-lg btn-primary-dark"
> > +          data-bind="click: previousStep">{% trans "Go Back & Fix Form" %}</button>
>
> i'm not sure, it might. i was just checking that you knew of it
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/12165/files/914d2442804f7e7f1157a6f891de7b457c4e8f11#r67910303>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAazfLUgXOfvjyQ3ilJtUn3P1pyHHdwpks5qOBsHgaJpZM4I6YaI>
> .
>